rpi/interrupts.go

```diff
@@ -86,7 +89,7 @@ func (pi *piPigpio) reconfigureInterrupts(ctx context.Context, cfg *Config) erro
 }
 
 // createNewInterrupt creates a new digital interrupt and sets it up with the specified configuration.
-func (ctx *reconfigureContext) createNewInterrupt(newConfig rpiutils.DigitalInterruptConfig, bcom uint) error {
+func (ctx *reconfigureContext) createNewInterrupt(newConfig rpiutils.PinConfig, bcom uint) error {
```

Comment:
I think we can get rid of the digital interrupt logic wholesale now, since drivers using interrupts have to access them using `DigitalInterruptByName`, which should set up the interrupt in this module. 

I think we can get rid of the interrupt tracking, and simply track mapping names to hardware pin numbers, the interrupt reconfigure code is kind of gathering dust right now.